php/utils.php

```diff
@@ -1737,30 +1737,16 @@ function random_name_generator() {
 /**
  * @param array  $assoc_args            Arguments array.
  * @param string $flag                  Flag to get the value.
- * @param array  $supported_flag_values array of supported flag values.
  * @param string $default_value         default flag value if flag is passed without value.
  *
  * @return string
  * @throws EE\ExitException
  */
-function get_value_if_flag_isset( $assoc_args, $flag, $supported_flag_values = [], $default_value = '' ) {
+function get_value_if_flag_isset( $assoc_args, $flag, $default_value = '' ) {
```

Comment:
Please make changes for this in all the packages where this is in use.
This function is being used in https://github.com/EasyEngine/site-type-php/ as well. It will break and give a fatal error if not handled.

Reply:
Done and updated PR description.